centreon/tests/e2e/cypress/e2e/Authentication/SAML/01-saml-authentication.feature

```diff
@@ -0,0 +1,19 @@
+Feature: SAML authentication
+    As an admin of Centreon Platform
```

Comment:
spacing seems off, please use the standard formatting

Reply:
done !